internal/setup/setup_test.go

```diff
@@ -1648,6 +1649,45 @@ func runParseReleaseTests(c *C, tests []setupTest) {
 	}
 }
 
+func (s *S) TestPackageMarshalYAML(c *C) {
+	for _, test := range setupTests {
+		c.Logf("Summary: %s", test.summary)
+
+		if test.relerror == "" || test.release == nil {
+			continue
+		}
+
+		data, ok := test.input["chisel.yaml"]
+		if !ok {
+			data = string(defaultChiselYaml)
+		}
+
+		dir := c.MkDir()
+		// Write chisel.yaml.
+		fpath := filepath.Join(dir, "chisel.yaml")
+		err := os.MkdirAll(filepath.Dir(fpath), 0755)
```

Comment:
Why?  We just got that directory out of MkDir three lines above.

Reply:
Removed.